.github/workflows/pysa.yml

```diff
@@ -0,0 +1,28 @@
+name: pysa
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  pysa:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+
+      - name: Install pyre-check
+        run: |
+          VERSION=$(grep "version" .pyre_configuration | sed -n -e 's/.*\(0\.0\.[0-9]*\).*/\1/p')
+          pip install pyre-check-nightly==$VERSION
```

Comment:
Could we build pysa from source instead?
The problem with using the latest nightly version is that the test might get stale, if an analysis change generates different results.

Reply:
@abishekvashok let's try again building from the source :) . The last time we had issue with typeshed not being picked up correctly but we can now debug this and make it work now that everything else works

Reply:
Okay on it!

Reply:
@r0rshark @arthaud when I try to compile pysa from source, I get a strange exception.

https://github.com/abishekvashok/pyre-check/runs/2960584609?check_suite_focus=true

```
ƛ Uncaught exception:

ƛ 

ƛ   Analysis__ClassHierarchy.Untracked("typing.NamedTuple")

ƛ 

ƛ Raised at file "command/analyzeCommand.ml", line 254, characters 6-17

ƛ Called from file "src/command.ml", line 2453, characters 8-238

ƛ Called from file "src/exn.ml", line 111, characters 6-10

ƛ Client exited with error code 1

Error: Process completed with exit code 2.
```

As wierd as it seems, I tried googling for the issue, and stumbled upon a stackoverflow answer which detailed it's due to typeshed being configured improperly, but if you view the log much above in the action, the stubs from typeshed directories are actually being recognised!

This is puzzling me a bit. I have extracted the typeshed.zip file in the action and even assigned the PYRE_TYPESHED evn variable but still :/

Is there anything else that needs to be done to configure typeshed? Is the python package build in the same manner?

Reply:
Solved the error. Will commit once I do a quick cleanup